helpers/helpers.go

```diff
@@ -282,6 +283,28 @@ func RunCommandSilent(cmdname string, args ...string) error {
 	return err
 }
 
+// RunCommandTimeout runs the given command with timeout + args and does not print command output
+func RunCommandTimeout(timeout int, cmdname string, args ...string) error {
+	ctx := context.Background()
+	// 0 means infinite timeout, ONLY set timeouts when value is > 0
```

Comment:
Maybe we want to throw an exception if the timeout is 0? Otherwise, this function call would effectively be something like RunCommandSilent.

Reply:
:rocket: